tcms/testplans/static/testplans/js/get.js

```diff
@@ -539,16 +539,52 @@ function toolbarEvents (testPlanId, permissions) {
         return false
     })
 
-    $('#default-tester-button').click(function (ev) {
-        $(this).parents('.dropdown').toggleClass('open')
+    $('input.user-field.typeahead').on('focusin', function () {
+        // Prevents sub-menu options from hidding when
+        // selecting typeahead suggestion.
+        $(this).parents('ul').css('display', 'block')
+    })
+
+    $('input.user-field.typeahead').typeahead({
+        minLength: 3,
+        highlight: true
+    }, {
+        name: 'default-tester-autocomplete',
+        // will display up to X results even if more were returned
+        limit: 100,
+        async: true,
+        display: function (element) {
+            return element.username
+        },
+        source: function (query, processSync, processAsync) {
+            jsonRPC('User.filter', { username__icontains: query }, function (data) {
```

Comment:
Note: `User.filter` depends on having the `auth.view_user` permission which currently silently prints errors in the console if the permission is not granted. 

I'm not sure if/how we want to inform users that their auto-complete searches may not work because they are missing the permission. 

Maybe we should default to the old implementation if permission isn't granted and allow auto-complete only when it is.